integration-cli/docker_cli_prune_unix_test.go

```diff
@@ -104,16 +105,16 @@ func (s *DockerDaemonSuite) TestPruneImageDangling(c *testing.T) {
 
 	out, err := s.d.Cmd("images", "-q", "--no-trunc")
 	assert.NilError(c, err)
-	assert.Assert(c, strings.Contains(strings.TrimSpace(out), id))
+	assert.Assert(c, is.Contains(strings.TrimSpace(out), id))
```

Comment:
Not necessarily for this PR, unless you need to make more changes; looks like these `strings.Trimspace()` calls here (and couple of other places below?) are redundant, as we're checking if it _contains_ `id`